docs/adding_software/overview.md

```diff
@@ -1,4 +1,24 @@
-# Adding software to EESSI
+# Overview of adding software to EESSI
+
+We welcome contributions to EESSI software stack. This page shows an overview of the procedures of contributing to the EESSI repository. 
```

Comment:
```suggestion
We welcome contributions to the EESSI software stack. This page shows the procedure and provides links to the contribution policy and the technical details of making a contribution. 
```

Reply:
Thanks for the suggestions. They all look good.